src/local/butler/run_bot.py

```diff
@@ -122,5 +123,8 @@ def _stop_handler(*_):
     signal.signal(signal.SIGTERM, _stop_handler)
     common.process_proc_output(proc)
     proc.wait()
+    exit_code = proc.wait()
```

Comment:
Does this mean to overwrite the previous line, instead of appending to it?

Reply:
Nope. that was a mistake.

Reply:
https://github.com/google/clusterfuzz/pull/3037